decidim-core/app/packs/src/decidim/sw/loader.js

```diff
@@ -1,8 +1,7 @@
 // check if the browser supports serviceWorker at all
 window.addEventListener("load", async () => {
   if ("serviceWorker" in navigator) {
-    // eslint-disable-next-line no-unused-vars
-    const registration = await navigator.serviceWorker.register("/sw.js", { scope: "/" });
+    await navigator.serviceWorker.register("/sw.js", { scope: "/" });
```

Comment:
Since you're not going to do anything with the `registration` variable, I think you can safely remove both `async/await` words